frontend/src/styles.css

```
@@ -87,6 +128,31 @@
+  --b2: #E8E4F8;
+  --b3: rgba(0, 0, 0, 0.12);
+  --b-focus: #4F6EF7;
+  --t1: #0D0F1A;
+  --t2: #3A3D52;
+  --t3: #6B7280;
+  --t4: #A0A8B8;
+  --t-placeholder: #B0B8CC;
+  --shadow-sm: 0 1px 3px rgba(0, 0, 0, 0.08), 0 1px 2px rgba(0, 0, 0, 0.06);
+  --shadow-md: 0 4px 12px rgba(0, 0, 0, 0.10), 0 2px 6px rgba(0, 0, 0, 0.06);
+  --shadow-lg: 0 24px 80px rgba(0, 0, 0, 0.12), 0 8px 24px rgba(0, 0, 0, 0.08);
+  color-scheme: light;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)